Presentation.Web/Tests/Helpers/GetDateHelper.ts

```diff
@@ -0,0 +1,22 @@
+﻿class GetDateHelper {
+
+    public static getTodayAsString() {
+        const currentDay = new Date().getDate();
+        const currentMonth = new Date().getMonth() + 1; // getMonth gets counts 0. So January is 0.
+
+        if (currentDay <= 9) {
```

Comment:
Det bliver noget rodet gentagelse... hvorfor ikke bygger hver komponent op for sig dvs. først bygger du "day". Så bygger du month og til sidst sætter du de tre komponenter sammen. Samtidig er logikken den samme så du kan lave en metode format(dayOrMonth : number) som laver det der trick med formattering. Og den formattering kan skrives som 

`("0" + dayOrMonth).slice(-2)`

Den sikrer at der altid er mindst 2 chars og hvis der er 3 f.eks. "012" så sikrer slice(-2) at det altid er de sidste 2 chars man får ud

Reply:
Den fixer jeg